snapcraft/plugins/catkin.py

```diff
@@ -25,6 +25,9 @@
     - catkin-packages:
       (list of strings)
       List of catkin packages to build.
+    - source-space:
```

Comment:
Is `source-space` the word used upstream, if not, it feels weird. Maybe just [`workspace`](http://wiki.ros.org/catkin/Tutorials/create_a_workspace)?